test/node-fetch/headers.js

```diff
@@ -279,4 +279,24 @@ describe('Headers', () => {
     // eslint-disable-next-line quotes
     expect(format(headers)).to.equal("{ a: [ '1', '3' ], b: '2', host: 'thehost' }")
   })
+
+  it('shoud accept headers as objects with array values', () => {
```

Comment:
Can we move this test to a non node-fetch suite? I'm worried we may overright this in the future if we ever resync with node-fetch.

Reply:
done :)